helpers/lib/fonts.js

```diff
@@ -84,58 +82,90 @@ const fontProviders = {
  * @returns {Object.<string, Array>|string}
  */
 module.exports = function(format, themeSettings, handlebars, options) {
-    
-    const collectedFonts = {};
-    _.each(themeSettings, function(value, key) {
-        //check that -font is on end of string but not start of string
-        const fontKeySuffix = '-font';
-        if (!key.endsWith(fontKeySuffix)) {
-            return;
-        }
+    const collectedFonts = Object.entries(themeSettings)
+        .reduce(
+            (
+                collectedFonts,
+                [key, value]
+            ) => {
+                //check that -font is on end of string but not start of string
+                const fontKeySuffix = '-font';
+                if (!key.endsWith(fontKeySuffix)) {
+                    return collectedFonts;
+                }
 
-        const splits = value.split('_');
-        const provider = splits[0];
+                const splits = value.split('_');
+                const provider = splits[0];
 
-        if (typeof fontProviders[provider] === 'undefined') {
-            return;
-        }
+                if (typeof fontProviders[provider] === 'undefined') {
+                    return collectedFonts;
+                }
+
+                if (typeof collectedFonts[provider] === 'undefined') {
+                    collectedFonts[provider] = [];
+                }
 
-        if (typeof collectedFonts[provider] === 'undefined') {
-            collectedFonts[provider] = [];
-        }
+                collectedFonts[provider].push(value);
 
-        collectedFonts[provider].push(value);
-    });
+                return collectedFonts;
+            },
+            {}
+        );
 
     // Parse font strings based on provider
-    const parsedFonts = _.mapValues(collectedFonts, function(value, key) {
-        return fontProviders[key].parser(value);
-    });
+    const parsedFonts = Object.entries(collectedFonts).reduce(
```

Comment:
It would be nice we can move this into a helper method with a callback to avoid duplication

Reply:
Agree, I prefer to avoid that level of refactor in this PR if thats ok with you, maybe we can do in future PR?